components/contact-details.tsx

```diff
@@ -60,7 +67,12 @@ export function ContactDetails({ contact }: ContactDetailsProps) {
         <DescriptionItem label="Keterangan" value={contact.keterangan} />
         <DescriptionItem label="Lokasi" value={contact.lokasi} />
         <DescriptionItem label="Kontak" value={contact.kontak} withCopyButton />
-        <DescriptionItem label="Alamat" value={contact.alamat} withCopyButton />
+        <DescriptionItem
```

Comment:
For the first iteration, I think it's still acceptable.
However, we should make it more consistent by putting it under the `Salin` button, so all action buttons are located on the right side.

@andriawan could you please create another issue to follow up on this?
Thanks!

Reply:
Issue created on #344